### PR TITLE
ENH: fix latex rendering in html + reduce white spaces + remove horiz…

### PR DESCRIPTION
**Pull request by agramfort**
ENH: fix latex rendering in html + reduce white spaces + remove horiz line in subopt relative

### CODE REVIEW

**Comment by tanglef** [comment]
I don't know if you are aware, but LaTeX rendering using Plotly is very unstable for now with Firefox.
Axis labels are sometimes (most of the times) not showing if rendered in TeX (but with the "ugly display" with dollars at least they were shown).
Several issues are opened about it in the Plotly repos.

**Comment by agramfort** [comment]
@tanglef I let you review. For me it's good to go.

File: benchopt/plotting/html/templates/benchmark.mako.html [hunk 1]
```diff
@@ -23,6 +23,9 @@
 <%
   ll_main = results[0]["sysinfo"]['main']
   need_filter = False
+  items_unique = dict()
+  for key, val in ll_main.items():
+    items_unique[key] = set([r['sysinfo']['main'][key] for r in results])
```
[the remark this answers is not in the document]
Comment (tanglef) [reply benchopt/plotting/html/templates/benchmark.mako.html:28]: Thank you for that, I didn't catch it

**Review by tanglef — APPROVED** [review]
All good for me !

File: benchopt/plotting/html/templates/benchmark.mako.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (tomMoral) [reply benchopt/plotting/html/templates/benchmark.mako.html:28]: could we sort them so we get some determinist rendering ?

Comment (tomMoral) [inline benchopt/plotting/html/templates/benchmark.mako.html:28]:
```suggestion
    all_items = list(set([r['sysinfo']['main'][key] for r in results]))
    items_unique[key] = sorted(all_items)

```

**Review by tomMoral — APPROVED** [review]
just a nitpick but other LGTM.

**Pull request merged**